pkgs/by-name/py/pywebcopy/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  fetchPypi,
+  python3,
+  python3Packages,
+}:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "pywebcopy";
+  version = "7.0.2";
+  pyproject = true;
+
+  src = fetchPypi {
```

Comment:
fetching from repo ( eg github) is preferred as published artifacts often don't include  tests

Reply:
@kirillrdy I also used the GitHub source but setup.py does not include tests. So I would keep `fetchPypi`.